assets/app/views/browse/pod.html

```diff
@@ -80,11 +80,15 @@
                 <annotations annotations="pod.metadata.annotations"></annotations>
               </tab>
 
-              <!--
-              <tab heading="Metrics" active="selectedTab.metrics">
-              Metrics content
+              <tab ng-if="metricsAvailable && pod.status.phase === 'Running'" heading="Metrics" active="selectedTab.metrics">
```

Comment:
wouldn't we still have the historical metrics for recently stopped pods?  i might have a job that ran and I want to check its cpu utilization or something for the time that it was running.